libs/chain-signatures/contract/Cargo.toml

```diff
@@ -30,6 +30,7 @@ k256 = { version = "0.13.4", features = [
     "arithmetic",
     "expose-field",
 ] }
+dcap-qvl = { git = "https://github.com/mattlockyer/dcap-qvl" }
```

Comment:
Some sketchy code changes there that might impact contract verification security. :/ Needs a closer look.

Reply:
Will be addressed by @near-bookrock in a separate PR (see: https://github.com/orgs/near/projects/175?pane=issue&itemId=113958602&issue=near%7Cmpc%7C451).